src/main/java/frc/robot/commands/IntakeDefaultCommand.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.math.filter.Debouncer;
+import edu.wpi.first.math.filter.Debouncer.DebounceType;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeDefaultCommand extends CommandBase{
```

Comment:
Why do we have both this and `CrocodileDefaultCommand`? I also see differences between this and the crocodile default, I'm not sure which one is correct. I will take care of refactoring later, for now please keep calling things crocodile.

Reply:
I forgot to delete that one, sorry.